www-servers/nginx-unit/nginx-unit-1.3-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="unit-${PV}"
+
+DESCRIPTION="A dynamic web and application server"
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="perl python ruby"
+REQUIRED_USE="|| ( ${IUSE} )"
+
+DEPEND="perl? ( dev-lang/perl:= )
+	python? ( dev-lang/python:= )
```

Comment:
Depending on Python directly is forbidden. Use `python-single-r1` to provide proper choice for Python version.

Reply:
These dependencies are only a way to get the user to choose at least one of the supported application languages. Please see [here](
https://github.com/gentoo/gentoo/pull/9249#discussion_r202655772) for an explanation I added to the initial ebuild. It does not matter which version of Python the user has selected, the build will use the one it finds.

I hope to be able to support multiple distinct Python versions in future versions, but as of now I only want to get Ruby support out.